app/templates/components/observation-progress-bar.hbs

```diff
@@ -0,0 +1,7 @@
+<div {{bind-attr class=':entry categoryScore'}}>
+  <div class="entry__date">{{shortDate dateObserved}}</div>
```

Comment:
I'm a bit concerned with each date/progress-bar/aqi entry lining up with their header since the widths for each column can be different for each row. A table might be more appropriate?